Fix: Add users Groups Parsing error feedback

fhconte opened the pull request:
## Proposed changes

New PR for task https://github.com/levante-framework/levante-dashboard/issues/187 
Now as small as it can be, and we can work on code improvements separately.

## Types of changes

What types of changes does this pull request introduce?

<!-- Put an `x` in the boxes that apply -->

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that does not add functionality but makes code cleaner or more efficient)
- [ ] Tests (new or updated tests)
- [ ] Styles (changes to code styling)
- [ ] CI (continuous integration changes)
- [ ] Other (please describe below)

Zio-4 commented:
Do we want to return right away when we can't find a group or scan all users and the provide a table of groups that are not found? I forgot how the surrounding code looks.

Zio-4 commented:
I just realized this is only for group. Can we do this for all the orgs? If you do site and school for example, it hangs with no error. Also, Can we change "district" to site? If you have a column called site it does not process it. When we send it to the backend however we still want to use district.

fhconte commented:
> I just realized this is only for group. Can we do this for all the orgs? If you do site and school for example, it hangs with no error. Also, Can we change "district" to site? If you have a column called site it does not process it. When we send it to the backend however we still want to use district.

Sent the correction. I also placed back activeSumbit = false as code documentation advices for...

Zio-4 reviewed (changes requested):
Looks good. I've decided we should check all users orgs and then show the error table. You can imagine a scenario where multiple rows have the wrong info then fixing those one by one, going back and forth between uploading it and fixing in the spread sheet would be very annoying.

fhconte commented:
@Zio-4 I added the sites error handling that we were missing and fixed ui behaviors after the error handling scenarios. 

Need some help to validate it and any possible different user cases we may have in the AddUsers process.

Zio-4 reviewed (changes requested):
Was able to add users for a cohort that does not exist. 
Also we should fix the org Ids part of the table. Not sure what it's supposed to be, so we can probably just delete it. 
![Screenshot 2025-05-02 at 10 57 56 AM](https://github.com/user-attachments/assets/8386c5b5-10dd-4500-ac25-51dbf53b7545)
